### ISSUE TITLE
Windows app crashes whenever I use WebView2

### ISSUE THREAD

**Issue by anacardix** [react-native-webview/react-native-webview#3323]
**Bug description:**
Implementing a WebView with the useWebView2 tag results in a crash without any error message.

```TS
function App(): React.JSX.Element {
  return (
    <SafeAreaView style={styles.container}>
      <MyWebComponent />
    </SafeAreaView>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  sectionContainer: {
    marginTop: 32,
    paddingHorizontal: 24,
  },
  sectionTitle: {
    fontSize: 24,
    fontWeight: '600',
  },
  sectionDescription: {
    marginTop: 8,
    fontSize: 18,
    fontWeight: '400',
  },
  highlight: {
    fontWeight: '700',
  },
});

const MyWebComponent = () => {
  return <WebView source={{uri: 'https://google.com/'}} useWebView2={true} />;
};
```

**To Reproduce:**
Simply run the application. Removing useWebView2 allows the app to work. Running it on macOS does not result in any issues.

**Environment:**
 - OS: Windows
 - OS version: 10 Enterprise 22H2
 - react-native: 0.73.4
 - react-native-webview: ^13.7.1
 - react-native-windows: 0.73.4

**Comment by tero-paananen** [comment]
Try enable `<WinUI2xVersion>2.8.1</WinUI2xVersion>` into `ExperimentalFeatures.props`

**Comment by anacardix** [comment]
> Try enable `<WinUI2xVersion>2.8.1</WinUI2xVersion>` into `ExperimentalFeatures.props`
> 
> https://github.com/react-native-webview/react-native-webview/blob/master/docs/Getting-Started.md#3-webview2-support

The issue has been resolved with react-native-webview version 13.8.1, independently of the <WinUI2xVersion>
